common/autoware_auto_perception_rviz_plugin/src/object_detection/detected_objects_display.cpp

```diff
@@ -57,6 +57,17 @@ void DetectedObjectsDisplay::processMessage(DetectedObjects::ConstSharedPtr msg)
       add_marker(label_marker_ptr);
     }
 
+    // Get marker for existence probability
+    auto existance_prob_marker = get_existence_probability_marker_ptr(
+      object.kinematics.pose_with_covariance.pose.position,
+      object.kinematics.pose_with_covariance.pose.orientation, object, object.classification);
```

Comment:
How about this in order to avoid multiple letters displayed on the same spot?

```cpp
    geometry_msgs::msg::Point existence_probability_position;
    existence_probability_position.x = object.kinematics.pose_with_covariance.pose.position.x + 0.5;
    existence_probability_position.y = object.kinematics.pose_with_covariance.pose.position.y;
    existence_probability_position.z = object.kinematics.pose_with_covariance.pose.position.z + 0.5;
    auto existance_prob_marker = get_existence_probability_marker_ptr(
      existence_probability_position,
      object.kinematics.pose_with_covariance.pose.orientation, object, object.classification);
```

Reply:
@kminoda Thank you for your review. I reflected your recommendation at https://github.com/autowarefoundation/autoware.universe/pull/5986/commits/6706a0ae7648c72f10505fe4ebb6a8648f057494
It looks much better 
![image](https://github.com/autowarefoundation/autoware.universe/assets/94814556/d9d61a2b-5d8d-4c83-9bbc-223101bd9f31)